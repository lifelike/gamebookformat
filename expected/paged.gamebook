title = Paged Option
id = paged

= Introduction
This gamebook is formatted with the [b]paged[/b] option.
Every section should begin at the top of a new page.
Not the introsections though.

= Another Heading
This is another introsection, just to confirm that
it is not cause a new page-break.

= Formats
Only some formats support page-breaks. HTML
for instance currently does not (although it
is technically possible to add it using
print stylesheets).

* 1 start
First section. You can go to [[second]] or [[third]]
from here.

* second
This is the second section. Yay.
You can go on to [[third]] or [[end]] or back to [[start]].

* third
This is the third section. It might or might not
be numbered 3 though, because of the automatic
section numbering in gamebookformat.
You can go on to the end at [[end]].

* end
This is The End.

