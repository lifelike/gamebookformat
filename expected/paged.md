# Paged Option #


## Introduction ##
 This gamebook is formatted with the paged option. Every section should begin at the top of a new page. Not the introsections though. 

## Another Heading ##
 This is another introsection, just to confirm that it is not cause a new page-break. 

## Formats ##
 Only some formats support page-breaks (HTML, RTF, LaTeX). 

Turn to 1 to begin.
<a name="section1">

## 1 ##
 First section. You can go to [4](#section4)
 or [3](#section3)
 from here. 
<a name="section2">

## 2 ##
 This is The End. 
<a name="section3">

## 3 ##
 This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at [2](#section2)
. 
<a name="section4">

## 4 ##
 This is the second section. Yay. You can go on to [3](#section3)
 or [2](#section2)
 or back to [1](#section1)
. 

