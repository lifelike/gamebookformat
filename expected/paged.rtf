{\rtf1\ansi\ansicpg1252\cocoartf1038\cocoasubrtf360
{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
{\colortbl;\red255\green255\blue255;}
\paperw11900\paperh16840\margl1440\margr1440\vieww14140\viewh14860\viewkind0
\pard\tx566\tx1133\tx1700\tx2267\tx2834\tx3401\tx3968\tx4535\tx5102\tx5669\tx6236\tx6803\ql\qnatural\pardirnatural

\f0\b\fs24 \cf0

\b \qc Paged Option
\b0\
\
\b \qc Introduction
\b0\



\ql  This gamebook is formatted with the paged option. Every section should begin at the top of a new page. Not the introsections though.  \
\
\
\b \qc Another Heading
\b0\



\ql  This is another introsection, just to confirm that it is not cause a new page-break.  \
\
\
\b \qc Formats
\b0\



\ql  Only some formats support page-breaks (HTML, RTF, LaTeX).  \
\
\b Turn to 1 to begin.
\b0\
\
\page \b \qc {\*\bkmkstart s1}1{\*\bkmkend s1}
\b0\
\ql  First section. You can go to \b {\field{\*\fldinst  REF s4 \\h }{\fldrslt 4}}
\b0
 or \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
 from here.  \
\

\page \b \qc {\*\bkmkstart s2}2{\*\bkmkend s2}
\b0\
\ql  This is The End.  \
\

\page \b \qc {\*\bkmkstart s3}3{\*\bkmkend s3}
\b0\
\ql  This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
.  \
\

\page \b \qc {\*\bkmkstart s4}4{\*\bkmkend s4}
\b0\
\ql  This is the second section. Yay. You can go on to \b {\field{\*\fldinst  REF s3 \\h }{\fldrslt 3}}
\b0
 or \b {\field{\*\fldinst  REF s2 \\h }{\fldrslt 2}}
\b0
 or back to \b {\field{\*\fldinst  REF s1 \\h }{\fldrslt 1}}
\b0
.  \
\

}
