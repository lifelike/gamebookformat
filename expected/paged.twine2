<tw-storydata name="Paged Option" startnode="1" creator="Twine" creator-version="2.0" format="Harlowe" options="">
<style role="stylesheet" id="twine-user-stylesheet" type="text/twine-css"></style><script role="script" id="twine-user-script" type="text/twine-javascript"></script>
<tw-passagedata pid="1" name="1" tags="" position="10,10"> First section. You can go to [[4]] or [[3]] from here. </tw-passagedata>
<tw-passagedata pid="2" name="2" tags="" position="20,20"> This is The End. </tw-passagedata>
<tw-passagedata pid="3" name="3" tags="" position="30,30"> This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at [[2]]. </tw-passagedata>
<tw-passagedata pid="4" name="4" tags="" position="40,40"> This is the second section. Yay. You can go on to [[3]] or [[2]] or back to [[1]]. </tw-passagedata>
</tw-storydata>
