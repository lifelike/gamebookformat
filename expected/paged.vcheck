examples/paged.json: ending (death?) section found: end
