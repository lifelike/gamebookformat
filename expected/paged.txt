    Paged Option

Introduction
 This gamebook is formatted with the paged option. Every section should begin at the top of a new page. Not the introsections though. 

Another Heading
 This is another introsection, just to confirm that it is not cause a new page-break. 

Formats
 Only some formats support page-breaks (HTML, RTF, LaTeX). 


Turn to 1 to begin.
1
 First section. You can go to 4 or 3 from here. 

2
 This is The End. 

3
 This is the third section. It might or might not be numbered 3 though, because of the automatic section numbering in gamebookformat. You can go on to the end at 2. 

4
 This is the second section. Yay. You can go on to 3 or 2 or back to 1. 

